operator/pkg/gateway-api/cell.go

```diff
@@ -77,8 +79,13 @@ func initGatewayAPIController(params gatewayAPIParams) error {
 		params.Logger.WithError(err).Error("Required GatewayAPI resources are not found, please refer to docs for installation instructions")
 		return nil
 	}
+	if helpers.CheckServiceImportCRD(context.Background(), params.CtrlRuntimeManager.GetClient()) {
+		params.Logger.Info("The ServiceImport CRD has been detected and thus its GatewayAPI integration enabled")
+	} else {
+		params.Logger.Info("The ServiceImport CRD has not been detectect and thus its GatewayAPI integration disabled")
```

Comment:
@MrFreezeex We should mention `MCS` in the log message somehow.

Personally i would change it to something like this

```
<nit: please a new-line between the checks>

helpers.CheckServiceImportCRD(context.Background(), params.CtrlRuntimeManager.GetClient())
params.Logger.
	WithField("enabled", helpers.HasServiceImportCRD()).
	Info("Multi-cluster Service API ServiceImport integration")
```

The context `Gateway API` is already given by the name of the logger subsystem. (see example message below)

```
level=info msg="Multi-cluster Service API ServiceImport integration" enabled=false subsys=gateway-api
```

In addition, this would allow us to get rid of the bool return value from `CheckServiceImportCRD` (-> because we have `HasServiceImportCRD` for this)